getting_started/workflow/export/exporting_pcks.rst

```diff
@@ -0,0 +1,129 @@
+original game loads in at runtime.
+
+Generating \*.pck files
+-----------------------
+
+In order to pack all resources of a project into a \*.pck file open the project
+and go to Project/Export and click on “Export PCK/Zip”. Also make sure to have
+an export template selected while doing so.
+
+.. image:: img/export_pck.png
+
+Another method would be to `export from the command line
+<https://docs.godotengine.org/en/latest/getting_started/editor/command_line_tutorial.html#exporting>`_.
+If the output file ends with a \*.pck or \*.zip file extension, then the export
+process will build that type of file for the chosen platform.
+
+.. note::
+   If one wishes to support mods for their game, they will need their users to
+   create similarly exported files. Assuming the original game expects a
+   certain structure for the \*.pck's resources and/or a certain interface for
+   its scripts, then either...
+
+   1. The developer must publicize documentation of these expected structures/
+      interfaces, expect modders to install Godot Engine, and then also expect
+      those modders to conform to the documentation's defined API when building
+      mod content for the game (so that it will work). Users would then use
+      Godot's built in exporting tools to create a \*.pck file, as detailed
+      above.
+   2. The developer uses Godot to build a GUI tool for adding their exact API
+      content to a project. This Godot tool must either run on a tools-enabled
+      build of the engine or have access to one (distributed alongside or
+      perhaps in the original game's files). The tool can then use the Godot
+      executable to export a \*.pck file from the command line with
+      :ref:`OS.execute <class_OS_execute>`. It makes the most sense for the
+      game to not use a tool-build though (for security) and for the modding
+      tools to *do* use a tool-enabled engine build.
+
+Opening \*.pck files at runtime
+-------------------------------
+
+To import a \*.pck file, one uses a one-liner. Keep in mind, there is no
+error or exception if the import fails. Instead, one might have to create some
+validation code as a layer on top. The following example expects a “mod.pck”
+file in the directory of the games executable. The \*.pck file contains a
+“mod_scene.tscn” test scene in its root.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    func _your_function():
+        ProjectSettings.load_resource_pack("res://mod.pck")
+        # Now one can use the assets as if they had them in the project from the start
+        var imported_scene = load("res://mod_scene.tscn")
+
+ .. code-tab:: csharp
+
+    public void _YourFunction()
+    {
+        ProjectSettings.load_resource_pack("res://mod.pck");
+        // Now one can use the assets as if they had them in the project from the start
+        var imported_scene = ResourceLoader.load("res://mod_scene.tscn") as PackedScene;
```

Comment:
C# API uses the naming case convention from C# where possible. This should be `ProjectSettings.LoadResourcePack` and `ResourceLoader.Load`. Also `importedScene`.